Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/DefaultLibrarySourceProvider.java

```diff
@@ -50,9 +50,8 @@ public boolean accept(File path, String name) {
                 if (indexOfVersionSeparator >= 0) {
                     Version version = new Version(fileName.substring(indexOfVersionSeparator + 1));
                     // If the file has a version, make sure it is compatible with the version we are looking for
-                    if (indexOfVersionSeparator == libraryName.length() && requestedVersion == null ||
-                            (version.isComparable(requestedVersion) && version.compatibleWith(requestedVersion))) {
-                        if (mostRecent == null || (version.isComparable(requestedVersion) && version.compareTo(mostRecent) > 0)) {
+                    if (indexOfVersionSeparator == libraryName.length() && requestedVersion == null || version.compatibleWith(requestedVersion)) {
+                        if (mostRecent == null || version.compareTo(mostRecent) > 0) {
```

Comment:
This can't use compareTo without checking isComparable, or it will throw. If both versions are not comparable, then it's based on strict equality only.

Reply:
Updated this!

Reply:
This change addresses the comparable case, but it's still not checking the strict equality case. If the versions aren't comparable, but they are equal then use that version.